cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilderTest.java

```diff
@@ -846,11 +844,11 @@ public void testWithTlsListenersWithCustomCerts()  {
                 .withType(KafkaListenerType.INTERNAL)
                 .withTls(true)
                 .withNewConfiguration()
-                    .withNewBrokerCertChainAndKey()
-                        .withSecretName("my-secret")
-                        .withKey("my.key")
-                        .withCertificate("my.crt")
-                    .endBrokerCertChainAndKey()
+                .withNewBrokerCertChainAndKey()
+                .withSecretName("my-secret")
+                .withKey("my.key")
+                .withCertificate("my.crt")
+                .endBrokerCertChainAndKey()
```

Comment:
This is an example of "less readable". We're building a tree of objects here and were using the indentation to reflect the tree structure. Can we revert this one and similar cases?

Reply:
thanks Tom, yes that make sense